### PR TITLE
Drop luet, image extractor, drop build code and multiarch images

### PR DESCRIPTION
**Pull request by Itxaka**
- drop luet
 - use image extractor for pulling images instead of luet
 - drop channel sources
 - introduce archfor images
 - drop build commands and useless code no needed
 - add pull-image command
 - bump yip to 1.x

Signed-off-by: Itxaka <itxaka.garcia@spectrocloud.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
This needs a docs update as we no longer support the auth flags for upgrade. 

We now support any type of docker auth that docker provides and it extends to anything using the pull, i.e. install,upgrade,pull-image

So it will look into:

~/.docker/config.json, or %USERPROFILE%\.docker\config.json by default
Docker config file location if  DOCKER_CONFIG is set
${XDG_RUNTIME_DIR}/containers/auth.json for podman

In that given order. So we need to document this after this is merged

**Comment by Itxaka** [comment]
docs PR here https://github.com/kairos-io/kairos-docs/pull/9

File: tests/mocks/extractor_mock.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	}
+	return &FakeImageExtractor{
+		Logger: logger,
+	}
+}
+
+func (f FakeImageExtractor) ExtractImage(imageRef, destination, platformRef string, local bool) error {
+	f.Logger.Debugf("extracting %s to %s in platform %s", imageRef, destination, platformRef)
+	if f.SideEffect != nil {
+		f.Logger.Debugf("running sideeffect")
```
Comment (mauromorales) [inline tests/mocks/extractor_mock.go:40]:
```suggestion
		f.Logger.Debugf("running side effect")
```

**Review by mauromorales — APPROVED** [review]
w00t w00t :clap:

File: pkg/action/install_test.go [hunk 2]
```diff
@@ -343,12 +343,11 @@ var _ = Describe("Install action tests", func() {
 
 		It("Fails if luet fails to unpack image", Label("image", "luet", "unpack"), func() {
```
Comment (jimmykarily) [inline pkg/action/install_test.go:344]: the description should no longer mention "luet" I guess

**Pull request merged**